app/src/layouts/appLayout/projectSidebar/organizationsControl/organizationsPopover/organizationsItem/organizationsItem.jsx

```diff
@@ -44,22 +44,60 @@ export const OrganizationsItem = ({
 }) => {
   const { formatMessage } = useIntl();
   const [isCollapsed, setIsCollapsed] = useState(isOpen);
+  const [isHoveredShowOpenButton, setIsHoveredShowOpenButton] = useState(false);
+  const [isFocusedShowOpenButton, setIsFocusedShowOpenButton] = useState(false);
+
   const ArrowIcon = isCollapsed ? ArrowDownIcon : ArrowRightIcon;
 
+  const onShowOpenButton = () => {
+    setIsFocusedShowOpenButton(true);
+  };
+
+  const onCloseOpenButton = () => {
+    setIsFocusedShowOpenButton(false);
+  };
+
+  const onCollapse = ({ target }) => {
```

Comment:
Let's simplify this by just resetting focus on click as discussed.

Reply:
Done.